core-toolbar.css

```diff
@@ -7,63 +7,86 @@ Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 */
 
-:host {
+core-toolbar {
   /* technical */
   display: block;
   position: relative;
   box-sizing: border-box;
   -moz-box-sizing: border-box;
+
   /* size */
-  height: 64px;
+  min-height: 64px;
+
   /* typography */
   font-size: 1.3em;
+
   /* background */
   background-color: #CFD8DC;
 }
 
-:host(.animate) {
+core-toolbar.animate {
   /* transition */
   transition: height 0.18s ease-in;
 }
 
-:host(.medium-tall) {
+core-toolbar.medium-tall {
   height: 128px;
 }
 
-:host(.tall) {
+core-toolbar.tall {
   height: 192px;
 }
 
-.toolbar-tools {
+core-toolbar .toolbar-tools {
   position: relative;
   height: 64px;
-  padding: 0 8px;
+  padding: 0 16px;
   pointer-events: none;
 }
 
+/* ******************************* */
+/* TODO: Use media queries? `core-narrow` is used in only one other component */
+/* ******************************* */
+
 /* narrow layout */
-:host(.core-narrow),
-:host-context(.core-narrow) {
+/*.core-narrow core-toolbar,
```

Comment:
If this isn't needed, we can just remove it. It will live on in source control :ghost: